tests/subworkflows/nf-core/align_bowtie2/test.yml

```diff
@@ -5,14 +5,14 @@
     - subworkflows/bam_sort_samtools
     - subworkflows/bam_stats_samtools
     # Modules
-    - bowtie2
-    - bowtie2/align
-    - samtools
-    - samtools/index
-    - samtools/sort
-    - samtools/stats
-    - samtools/idxstats
-    - samtools/flagstat
+    # - bowtie2
```

Comment:
Would rather these lines are deleted rather than commented out

Reply:
Not opposed, but it would be more to go track these down later. But I'm not as scared now, I think we have a lot of hands to mobilize and can add all of these in quickly.